packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -356,7 +372,14 @@ export function getContractCallOperations({
   });
 }
 
-/** @hidden */
+/**
+ * Extracts a transfer operation from a transaction receipt, determining the addresses and types
```

Comment:
```suggestion
 * Extracts a transfer operation from a transaction receipt, determining the addresses and types
 *
```

Reply:
I canny see this check in the commit 👀

Reply:
Tbf I think I made this suggestion on the wrong line :S